shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -173,6 +173,14 @@ - (NSDictionary*)getClipboardData:(NSString*)format;
  */
 - (void)setClipboardData:(NSDictionary*)data;
 
+/**
+ * Returns true iff the clipboard contains nonempty string data.
+ *
+ * See also:
+ *   * https://developer.apple.com/documentation/uikit/uipasteboard/1829416-hasstrings
```

Comment:
Why are you referencing UIKit documentation in the macOS embedding?

Reply:
There seems to be no equivalent hasStrings method for Mac.  I'll add some details in the comment.